lib/models/inquiry_form_data.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'inquiry_form_data.freezed.dart';
+part 'inquiry_form_data.g.dart';
+
+@freezed
+class InquiryFormData with _$InquiryFormData {
+  const factory InquiryFormData({
+    required bool enabled,
+    required String title,
+    required String content,
+  }) = _InquiryFormData;
+
+  factory InquiryFormData.empty() => const InquiryFormData(
+        enabled: false,
+        title: '고객센터 문의합니다.',
+        content: '',
+      );
```

Comment:
생성자보단 static const 객체로 선언하는 것이 더 적절해 보입니다.

```suggestion
  static const empty = InquiryFormData(
    enabled: false,
    title: "고객센터 문의합니다.",
    content: "",
  );
```

다만, 별도의 객체를 선언하는 것보단 provider에서 초기화하는 것이 흐름이 더 매끄러울 것 같습니다.